charts/kube-acp-stack/Chart.yaml

```diff
@@ -7,7 +7,7 @@ sources:
   - https://github.com/cloudentity/acp-helm-charts
 dependencies:
 - name: hazelcast
-  version: "3.7.1"
+  version: "2.11.0"
```

Comment:
Is that correct?
Is the chart 2.11 pointing to hz 3.12?

Reply:
Yes, please take a look at: https://hazelcast-charts.s3.amazonaws.com/index.yaml